pkg/handler/loki.go

```diff
@@ -145,11 +141,55 @@ func GetFlows(cfg LokiConfig) func(w http.ResponseWriter, r *http.Request) {
 			writeError(w, http.StatusServiceUnavailable, msg)
 			return
 		}
-		hlog.Infof("GetFlows raw response: %v", string(resp))
+		hlog.Debugf("GetFlows raw response: %s", resp)
 		writeRawJSON(w, http.StatusOK, resp)
 	}
 }
 
+func selectTimeRange(w http.ResponseWriter, param string, extraArgs *strings.Builder) {
+	r, err := strconv.ParseInt(param, 10, 64)
+	if err != nil {
+		writeError(w, http.StatusServiceUnavailable, err.Error())
```

Comment:
I know this is old code, but, I would put a different status than "Service Unavailable", no? Shouldn't it be something like 400 / BadRequest here  ?